server/globals.ts

```diff
@@ -16,3 +16,48 @@ declare let IPTools: typeof import('./ip-tools').IPTools;
 declare let Sockets: typeof import('./sockets');
 // let TeamValidator: typeof import('../sim/team-validator');
 declare let TeamValidatorAsync: typeof import('./team-validator-async');
+
+interface Group {
+	symbol: string,
+	id: string,
+	name: string,
+
+	root?: boolean,
+	inherit?: string,
+	jurisdiction?: string,
+	roomonly?: boolean,
+	globalonly?: boolean,
```

Comment:
```suggestion
	globalonly?: boolean,

```

(Separate the permission metadata above from the permissions list below.)

Reply:
```suggestion
	/** `true` to disallow room-promoting users to this group. */
	globalonly?: boolean,

```